test/tests/Path.js

```diff
@@ -611,3 +611,13 @@ test('Path#arcTo(from, through, to); where from, through and to all share the sa
     }
     equals(error != null, true, 'We expect this arcTo() command to throw an error');
 });
+
+test('#1493 Path#add with 1000000 segments', function() {
+    var path = new Path();
+    for (var i=0; i<1000000; i++)
+    {
```

Comment:
Could you bring the bracket to the same line as the `for`?  That's the convention in the paper.js code-base

Reply:
I added this one too.